debian/changelog

```diff
@@ -1,3 +1,9 @@
+confluent-ksqldb (0.17.0-1) unstable; urgency=medium
```

Comment:
These changes got pulled in because I made this branch from the 0.17.0-ksqldb tag I think, are these changes okay to have in this branch? the 0.17.0-ksqldb _branch_ was cut from 6.2.x which is why it doesn't already have these. WDYT @JimGalasyn @vcrfxia

Reply:
@lct45, these `*-ksqldb` branches are used only for docs, so I think this is fine.